cas-server-core-webflow/src/main/java/org/apereo/cas/web/flow/AbstractCasWebflowConfigurer.java

```diff
@@ -68,12 +61,10 @@
  * @author Misagh Moayyed
  * @since 4.2
  */
-@RefreshScope
-@Component("casWebflowConfigurer")
-public abstract class AbstractCasWebflowConfigurer {
-    private static final String FLOW_ID_LOGIN = "login";
+public abstract class AbstractCasWebflowConfigurer implements CasWebflowConfigurer {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This class has too many methods, consider refactoring it.](https://www.codacy.com/app/mmoayyed/cas/file/2749420172/issues/source?bid=3256115&fileBranchId=3357337#l64)